src/SprykerEco/Yves/FactFinderWebComponents/Theme/default/components/atoms/campaign/campaign.twig

```diff
@@ -0,0 +1,25 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'campaign',
+    tag: 'campaign'
```

Comment:
Why do we need a custom element here?

Reply:
It is not required and I removed from all molecules and atoms custom tag